modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
+
+[[removed-or-deprecated-plugins]]
+=== Removed or deprecated plugins
+
+[[bbcode]]
+==== BBCode
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `bbcode` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
+
+[[color-picker]]
+==== Color Picker
+
+Color Picker’s functionality has been a built in part of {productname} since version 5.
+
+Consequently, and as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the empty `colorpicker` plugin has been removed.
+
+[[context-menu]]
+==== Context Menu
+
+Context Menu’s functionality has been a built in part of {productname} since version 5.
+
+Consequently, and as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the empty `contextmenu` plugin has been removed.
+
+[[full-page]]
+==== Full Page
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `fullpage` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
+
+[[hr]]
+==== HR
+
+The `hr` (horizontal rule) functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `hr` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[changed-plugins-imagetools]]
+==== ImageTools
+
+The `imagetools` plugin is no longer part of the Free {productname} editor. _Image Tools_ is now _Edit Image_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+[[legacy-output]]
+==== Legacy Output
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `legacyoutput` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
+
+[[changed-plugins-paste]]
+==== Paste
+
+The `paste` functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+Because the functionality was unmaintained, `paste` no longer supports input from Microsoft Word.
+
+There is an open https://github.com/tinymce/tinymce/discussions/7487[request for maintainers] should someone in the community be interested in taking over maintainance of this particular functionality as a separate plugin.
+
+If a community-maintained version of the `paste` plugin becomes available, we will link to it from the xref:migration-from-5x.adoc#plugins-paste[_Migration Guide_].
+
+NOTE: the Premium plugin, https://tiny.cloud/docs/plugins/premium/powerpaste/[`PowerPaste`], is available. It provides the capability to accept data from Microsoft Word and Microsoft Excel, and clean-up the received data before pasting it into place.
```

Comment:
```suggestion
NOTE: The Premium plugin, https://tiny.cloud/docs/plugins/premium/powerpaste/[`PowerPaste`], is available. It provides the capability to accept data from Microsoft Word and Microsoft Excel, and clean-up the received data before pasting it into place.
```

Reply:
```suggestion
NOTE: The Premium plugin, xref:introduction-to-powerpaste.adoc[`PowerPaste`], is available. It provides the capability to accept data from Microsoft Word and Microsoft Excel, and clean-up the received data before pasting it into place.
```

We should be linking to the current docs, not TinyMCE 5 docs.